RecoTracker/IterativeTracking/python/InitialStepPreSplitting_cff.py

```diff
@@ -128,9 +128,9 @@
 initialStepChi2EstPreSplitting = RecoTracker.MeasurementDet.Chi2ChargeMeasurementEstimator_cfi.Chi2ChargeMeasurementEstimator.clone(
     ComponentName = cms.string('initialStepChi2EstPreSplitting'),
     nSigma = cms.double(3.0),
-    MaxChi2 = cms.double(30.0),
+    MaxChi2 = cms.double(16.0),
     clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutLoose')),
-    pTChargeCutThreshold = cms.double(15.)
+#    pTChargeCutThreshold = cms.double(15.)
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant